packages/core/contracts/insured-bridge/BridgePool.sol

```diff
@@ -269,7 +269,8 @@ contract BridgePool is Testable, BridgePoolInterface, ERC20, Lockable {
         require(
             depositData.slowRelayFeePct <= 0.25e18 &&
                 depositData.instantRelayFeePct <= 0.25e18 &&
-                realizedLpFeePct < 0.5e18
+                realizedLpFeePct < 0.5e18,
```

Comment:
side comment: should `realizedLpFeePct` not be `<=`?

Reply:
Hmmmmm I think it should I'll change